article_parser.py

```diff
@@ -42,39 +42,41 @@ def character_recognition(self, text):
         text = phrase2.sub('.', text)
         text = phrase3.sub('', text)
         text = text.split(".") #распарсили текст
-        self.useful_operations(text)
+        self.pre_operations(text)
         self.critere_parser()
         return self.probable_names
 
     def critere_parser(self):
-        for index in range(len(self.sentences_list)): #имеет смысл иметь зип заглавных всех, заглавных первых (так будет быстрее)
+        def estimator(cur_word, num_word):
```

Comment:
Но вообще да, фигня полная, приберу ее
--
Envoyé de l'application Mail.Ru pour Android vendredi, 09 septembre 2016, 03:47PM +03:00 de Cadovvl < notifications@github.com> :

>In  article_parser.py :
>>          self.critere_parser()
>          return self.probable_names
>  
>      def critere_parser(self):
> -        for index in range(len(self.sentences_list)): #имеет смысл иметь зип заглавных всех, заглавных первых (так будет быстрее)
> +        def estimator(cur_word, num_word):
>Эту кучу вложенных if-ов читать невозможно. Когда обнаружится, что тут есть ошибка - ты ее никогда не найдешь. А понимать свой код перестанешь уже через пару недель.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .